### PR TITLE
Fix warning in code gen

### PR DESCRIPTION
**Pull request by kimdv**

### CODE REVIEW

File: CodeGeneration/Package.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ let package = Package(
     .executable(name: "generate-swiftsyntax", targets: ["generate-swiftsyntax"])
   ],
   dependencies: [
-    .package(url: "https://github.com/apple/swift-syntax.git", revision: "94b9021a2e461fc9a4b3bda6f4734119b52e8094"),
+    .package(url: "https://github.com/apple/swift-syntax.git", revision: "55349f8019fb129781af3f988011b20b66989664"),
```
Comment (kimdv) [inline CodeGeneration/Package.swift:14]: Also updated the swift-syntax version. Even there is noting new, just so we don't get too far behind.

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
@ahoppen I'm not sure if the new generated files is a mis in a previous PR?

But can we, some how, avoid missing code generation in the future? 
You mentioned the Ci **don't** have access to the internet. 

But can we, in the CI refer to a local swift-syntax package? 
I think the builder is now, OK stable?

**Review by ahoppen — APPROVED** [review]

**Pull request merged**